src/Controller/Frontend/TaxonomyController.php

```diff
@@ -40,13 +39,12 @@ public function __construct(Config $config, Environment $twig, TemplateChooser $
     public function listing(ContentRepository $contentRepository, Request $request, string $taxonomyslug, string $slug): Response
     {
         $page = (int) $request->query->get('page', 1);
+        $amountPerPage = $this->config->get('general/listing_records');
 
         /** @var Content[] $records */
-        $records = $contentRepository->findForListing($page);
+        $records = $contentRepository->findForTaxonomy($page, $taxonomyslug, $slug, $amountPerPage);
```

Comment:
Could $slug be renamed to something more meaningful? Like $itemSlug, $tagSlug or something like this?

Reply:
It's been `/{taxonomyslug}/{slug}` since Bolt 1, and I've never heard people being confused about it.. I'd prefer to keep simply `$slug`.